Blazor/README.md

```diff
@@ -202,10 +207,17 @@ Example:
 This function is mainly used by the `WetContainer` component and typically would not be called directly, but is begin made available should there be a need.
```

Comment:
This function seems to be javascript only, the description should reflect that but I think we should go even further:  We should probably create a .NET function that calls it (because we were able so far to keep users from having to do the "invokeasync" themselves.   Function could go in `utils.CDTSMain` class.